app/models/space.rb

```diff
@@ -0,0 +1,38 @@
+# This class helps deliver on the "Authorization System: use case 1-1".  In upcoming use cases we
+# will begin to look toward multiple spaces.  For the moment, let's use this narrow definition to
+# move us forward.
+#
+# @see https://github.com/orgs/forem/projects/46
+#
+# @note This class exists to assist with authorization and exposing an "ActiveModel" compliant
+#       interface for form work.
+class Space
+  # ...the final frontier.
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :limit_post_creation_to_admins, :boolean, default: lambda {
+                                                                 FeatureFlag.enabled?(:limit_post_creation_to_admins)
+                                                               }
+
+  # A convenience method to maintain Ruby idioms
+  alias limit_post_creation_to_admins? limit_post_creation_to_admins
+
+  DEFAULT = "default".freeze
+
+  # @note Right now there's only one space..."default"; this is used to generate the URL for our update.
+  def to_param
+    DEFAULT
+  end
+
+  def save
+    if limit_post_creation_to_admins?
+      FeatureFlag.enable(:limit_post_creation_to_admins)
+    else
+      FeatureFlag.disable(:limit_post_creation_to_admins)
+    end
+
+    # I want to ensure that we're returning true, to communicate that the "save" was successful.
+    true
+  end
```

Comment:
This in particular is what I'm referring to here: https://github.com/forem/forem/pull/16897#pullrequestreview-914488577

We likely wouldn't need this model at all if we create a feature flags controller, right? (if I'm understanding correctly 😅)

Reply:
Both yes and no.  I had this very thought.  And we do have a Feature Flag controller in the works.  However, in discussion with @anujbhavsar96 and @amywtlin the design for the spaces page is to incrementally introduce the concept.

So yes, we could handle this with the feature flag controller.  (And boy I wish we would) but for this moment in time, having them separate provides some incremental design approaches to rollout the product.

Reply:
And too be clear, good catch on the "hey one of these things is **so much** like the other."